package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "arabic-mathjax",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

```suggestion
  "version": "2.0.1",
```

Probably just a simple bump would do, or whatever https://semver.org/ thinks is ok.